src/main/java/core/basesyntax/service/WriteService.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface WriteService {

```

Comment:
let's rename the interfaces according to the principle mentioned by Lilya, for example ReportGeneratorService DataProcessorService and the corresponding implementations with the suffix Impl